src/index.html

```diff
@@ -48,6 +48,16 @@ <h2 class="title fw-bold mb-3">Add your score</h2>
           </button>
```

Comment:
### Optional
- [ ] It would be better if we add the `.vscode` to the `.gitignore` file